profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
```

Comment:
My assumption is that it's not possible to list all Profiles.  So, I'd expect us to document that `GET /api/eperson/profiles` is unsupported, and that you have to access the profile via the EPerson ID.

I'd also assume that the `/api/eperson/profiles` endpoint would NOT function at all if the Person Entity doesn't exist.  It seems like this endpoint requires Person Entities to be enabled.  Since Person entities are disabled by default, then we should ensure that, in that scenario, either the `profiles` endpoint should disappear from the REST API, or it should return always return a 404 or similar.

Reply:
I will document that GET /api/eperson/profiles is unsupported and that the entire repository is not enabled in case the "profile" entity (usually Person) is not defined on the system.